### DISCUSSION TITLE
/About page content

### DISCUSSION THREAD

**Discussion by b-bayport** [theatl-social/theatl-social-content#11] [Ideas]
Info about "deletion"

I think it would be good to add info about how deletion works on Mastodon to the theATL.social/about page. 

Something like this:

> Please remember that anything you post publicly on a Mastodon/Fediverse instance, really is public and once it propagates to other instances, you have no real control over it, especially that you may not be able to delete your content from other instances. The ActivityPub protocol specifies a server SHOULD delete content when the server of origin deletes it but it doesn’t HAVE TO and there’s no way to actually enforce the deletion. w3.org/TR/activitypub/#delete-...

   
Reference: https://strangeobject.space/@esther/109654308271487532

**Comment by mikehikes** [comment]
Privacy policy was re-written to reflect your note and emphasize that there is no actual "deletion" on the Fediverse
